### ISSUE TITLE
No improvement in validation loss

### ISSUE THREAD

**Issue by rahul545** [experiencor/keras-yolo2#431]
I am trying to train the model to detect empty spaces(non-object areas) in the images. But while training my validation loss is not reducing.
